.binstar.yml

```diff
@@ -0,0 +1,47 @@
+package: dask
+user: blaze
+
+platform: 
+  - linux-64
+  - linux-32
+  - osx-64
+  - win-32
+  - win-64
+
+engine:
+  - python=2.6
+  - python=2.7
+  - python=3.3
+  - python=3.4
+
+before_script:
+  - python -V
+
+install:
+  - conda config --add channels blaze
+
+script:
+  - conda build conda.recipe
+
+env:
+  - LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
```

Comment:
Probably necessary for handling strange formats in some database.  Suggest removal